src/docs/stan-reference/distributions.tex

```diff
@@ -1911,13 +1911,12 @@ \subsubsection{Probability Density Function}
 \distro{Wiener}(y|\alpha, \tau, \beta, \delta)
 =
 \frac{\alpha}{(y-\tau)^3/2}
-\,
+%
```

Comment:
No change---just reformatting so that there weren't overfull hboxes.  Also a bunch of other changes to add spaces to the code when I took a pass replacing if_else.